pyproject.toml

```diff
@@ -8,22 +8,21 @@ packages = [{ include = "scripts" }, { include = "tests" }]
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.10"
-cairo-lang = "0.13.0"
-amarna = "^0.1.5"
 starknet-py = "^0.20.0"
-py-evm = "^0.7.0a1"
-openzeppelin-cairo-contracts = "^0.6.1"
-ethereum = { git = "https://github.com/ethereum/execution-specs.git" }
+cairo-lang = "0.13.0"
+python-dotenv = "^0.21.0"
+async-lru = "^2.0.4"
+pandas = "^1.5.1"
+toml = "^0.10.2"
+matplotlib = "^3.8.2"
```

Comment:
I don't think that we need matplotlib nor pandas, and if it's the case, then you should probably update some imports so that it's not. Surprised by toml as well